files/zh-cn/web/media/formats/image_types/index.md

```diff
@@ -1131,66 +1203,67 @@ WebP now has broad support in the latest versions of major web browsers, althoug
               <th scope="col">Safari</th>
             </tr>
             <tr>
-              <th scope="row">Lossy WebP support</th>
+              <th scope="row">有损 WebP 支持</th>
               <td>17</td>
               <td>18</td>
               <td>65</td>
               <td>no</td>
-              <td>11.10 (Presto)<br />15 (Blink)</td>
-              <td>no</td>
+              <td>11.10（Presto）<br />15（Blink）</td>
+              <td>14</td>
             </tr>
             <tr>
-              <th scope="row">Lossless WebP</th>
-              <td>23<br />25 on Android</td>
+              <th scope="row">无损 WebP</th>
+              <td>Android 23<br />25</td>
               <td>18</td>
               <td>65</td>
-              <td>no</td>
-              <td>12.10 (Presto)<br />15 (Blink)</td>
-              <td>no</td>
+              <td>否</td>
+              <td>12.10（Presto）<br />15（Blink）</td>
+              <td>14</td>
             </tr>
             <tr>
-              <th scope="row">动画</th>
+              <th scope="row">动画/th>
               <td>32</td>
               <td>18</td>
               <td>65</td>
-              <td>no</td>
-              <td>19 (Blink)</td>
-              <td>no</td>
+              <td>否</td>
+              <td>19（Blink）</td>
+              <td>14</td>
             </tr>
           </tbody>
         </table>
+        <p>WebP 还可用于从 Firefox 96 和 Chrome 50 的画布中<em>导出</em>图像。（参见 <a href="/zh-CN/docs/Web/API/HTMLCanvasElement/toBlob#浏览器兼容性"><code>HTMLCanvasElement.toBlob()</code></a> 以获取更详细的支持情况）</p>
       </td>
     </tr>
     <tr>
-      <th scope="row">最高分辨率</th>
-      <td>16,383×16,383 pixels</td>
+      <th scope="row">最大尺寸</th>
+      <td>16,383×16,383 像素</td>
     </tr>
     <tr>
       <th scope="row">支持的颜色模式</th>
       <td>
-        Lossy WebP stores the image in 8-bit Y'CbCr 4:2:0 (YUV420) format.
-        Lossless WebP uses 8-bit ARGB color, with each component taking 8 bits
-        for a total of 32 bits per pixel.
+        有损 WebP 以 8 位 Y'CbCr 4:2:0 (YUV420) 格式存储图像。无损 WebP 使用 8 位 ARGB 颜色，每个分量 8 位，每个像素共 32 位。
       </td>
     </tr>
     <tr>
       <th scope="row">压缩</th>
-      <td>Lossless (Huffman, LZ77, or color cache codes) or lossy (VP8).</td>
+      <td>无损（Huffman、LZ77 或 color cache codes）或有损（VP8）</td>
     </tr>
     <tr>
       <th scope="row">授权</th>
-      <td>No license required; source code is openly available.</td>
+      <td>无需许可证；源代码可公开获取。</td>
     </tr>
   </tbody>
 </table>
 
-### <a id="XBM" name="XBM">XBM</a> (X Window System Bitmap file)
+> **备注：** 尽管 Apple [宣布在 Safari 14 中支持 WebP](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174)，但在 Safari 版本 16.0 之前，`.webp` 图像无法在早于 Big Sur 版本的 macOS 桌面版本上正常显示，而 iOS 14 上的 Safari *却能*正常显示 `.webp` 图像。
```

Comment:
wait for https://github.com/mdn/content/pull/30039/files to decide what content should be written here

Reply:
```suggestion
> **备注：** 在 macOS 版 Safari 上，WebP 支持取决于 Safari 和 macOS 的版本。你需要 Safari 14 或更高版本，以及 macOS Big Sur (11) 或更高版本。
```